calliope/backend/pyomo/variables.py

```diff
@@ -92,4 +92,5 @@ def initialize_decision_variables(backend_model):
 
     if model_data_dict['attrs'].get('run.ensure_feasibility', False):
         backend_model.unmet_demand = po.Var(backend_model.loc_carriers, backend_model.timesteps, within=po.NonNegativeReals)
+        backend_model.excess_supply = po.Var(backend_model.loc_carriers, backend_model.timesteps, within=po.NegativeReals)
```

Comment:
I wonder if there's a better name than ``excess_supply`` given that its negative, and an excess implies something >0. Doesn't matter hugely though

Reply:
I wondered about changing them to `demand_slack` and `supply_slack` and then generalising it to `slack` in the results Dataset, but dropped that idea as `slack` is perhaps unintuitive outside the optimisation field. `unused_supply` also came to mind. I'll have a think

Reply:
``unused_supply`` seems like it fits the bill

Reply:
Updated in df65948